packages/rke2-calico/generated-changes/patch/templates/crs/custom-resources.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/templates/crs/custom-resources.yaml
++++ charts/templates/crs/custom-resources.yaml
+@@ -6,6 +6,12 @@
+ {{ $secrets = append $secrets $item }}
+ {{ end }}
+ {{ $_ := set $installSpec "imagePullSecrets" $secrets }}
++{{ $networkSpec := get $installSpec "calicoNetwork" }}
++{{ $ipPools := get $networkSpec "ipPools" }}
++{{ $defaultipPools := first $ipPools }}
++{{ $defaultCIDR := get $defaultipPools "cidr" }}
++{{ $finalCIDR := coalesce .Values.global.clusterCIDR $defaultCIDR }}
```

Comment:
I'm quite concerned about necessity of using so much `get` statements just to access a nested value, but I'm not a Helm expert and I can't propose anything better. I would be happy if someone has a better idea though.

Still approving this PR.

Reply:
I know, I haven't found a better way but it might exist. Perhaps @brandond know

Reply:
We can probably come up with a more concise way to do this, I'll take a look this evening.

Reply:
Found a nicer approach :)

Reply:
Found a nicer approach :)